index.bs

```diff
@@ -190,19 +187,45 @@ Compute the layout shift {#sec-compute-layout-shift}
         1. Set <var>maxShiftFraction</var> to <var>shiftFraction</var> if <var>tuple</var>’s <var>shiftFraction</var> > <var>maxShiftFraction</var>.
     1. Subtract from <var>impactedRegion</var> all points that lie outside of the <a>viewport</a>.
     1. Let <var>impactedFraction</var> be the area of the <var>impactedRegion</var> divided by the area of the current <a>viewport</a>.
-    1. Set <var>layoutShift</var> to the <var>impactedFraction</var> multiplied by the <var>maxShiftFraction</var>.  
+    1. Set <var>layoutShift</var> to the <var>impactedFraction</var> multiplied by the <var>maxShiftFraction</var>.
     1. If <var>layoutShift</var> is greater than 0:
         1. Create a new {{LayoutShift}} object |newEntry|.
         1. Set |newEntry|'s {{PerformanceEntry/name}} attribute to <code>"layout-shift"</code>.
-        1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to <code>"layoutShift"</code>.
+        1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to <code>"layout-shift"</code>.
         1. Set |newEntry|'s {{PerformanceEntry/startTime}} attribute to <a>current high resolution time</a>.
         1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to 0.
-        1. Set |newEntry|'s {{LayoutShift/value}} attribute to <var>layoutShift</var>.
+        1. Set |newEntry|'s <dfn attribute for=LayoutShift>value</dfn> attribute to <var>layoutShift</var>.
+        1. Set |newEntry|'s <dfn attribute for=LayoutShift>lastInputTime</dfn> attribute
+            to the time of the most recent <a>excluding input</a>, or 0 if no excluding input has occurred
+            during the browsing session.
+        1. Set |newEntry|'s <dfn attribute for=LayoutShift>hadRecentInput</dfn> attribute
+            to <code>true</code> if {{LayoutShift/lastInputTime}} is less than 500 milliseconds in the past,
+            and <code>false</code> otherwise.
         1. <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">Queue the PerformanceEntry</a> |newEntry| object.
 </div>
 
 NOTE: This computation ensures that the layout shift takes into account both the fraction of the viewport that has been impacted by layout stability as well as the greatest impact to any given element in the viewport.  This is to recognize that a large element that moves only a small distance can have a small impact on the perceived stability of the page.
 
+Input Exclusion {#sec-input-exclusion}
+--------------------------------------
+
+An <dfn export>excluding input</dfn> is any event from an input device which signals a user's
+active interaction with the page, or any event which directly changes the size of the <a>viewport</a>.
```

Comment:
It's unclear here what the scope is because of the wording. Presumably if an input is directed at the main frame then an iframe does not count it as excluding input? Maybe changing 'page' to Window/Document would clarify this a bit.

Reply:
Changed to "document".